com.unity.shadergraph/Editor/GraphUI/DataModel/GraphDataNodeModel.cs

```diff
@@ -170,6 +170,8 @@ protected override void OnDefineNode()
             // TODO: Convert this to a NodePortsPart maybe?
             foreach (var portReader in nodeReader.GetPorts())
             {
+                if (portReader.GetField("IsStatic", out bool isStatic) && isStatic) continue;
```

Comment:
Static GraphDelta ports don't need GTF port models because they don't get their values from incoming connections. Instead, they're settings directly on the node, so we add parts for them in the graph UI element.

Reply:
Ahhh I see, thanks for that explanation!